pppd/pppd.8

```diff
@@ -298,6 +298,9 @@ to \fIn\fR seconds (default 3).
 .B chap-timeout \fIn
 Set timeout for CHAP authentication by peer to \fIn\fR seconds (default 60).
 .TP
+.B chapms-strip-domain
```

Comment:
```suggestion
.B chapms\-strip\-domain
```
dash character should be escaped in the TROFF language

Reply:
right, fixed